### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by nejidevelops**
In this PR, I implemented stylings using Vanilla CSS. 

Here is a screenshot of the recommended changes implemented

Screenshot from 2023-08-26 23-58-26

### CODE REVIEW

File: src/styles/styles.css [hunk 1]
```diff
@@ -1,5 +1,292 @@
...
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.circle {
+  margin-right: 10%;
+}
```
Comment (abdulhamiid) [inline src/styles/styles.css]: Zeplin
image

Your design
image

- [ ] Please add the `border` in the user icon to match Zeplin's design, 
- [ ] According to the project requirements, our final design should match Zeplin's design (this includes all elements + the same styling) 🤗🛠

image

File: src/styles/styles.css [hunk 2]
```diff
@@ -1,5 +1,292 @@
...
+.form-title {
+  font-family: var(--Montserrat);
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
```
Comment (abdulhamiid) [inline src/styles/styles.css:251]: Original design
bookstore

Your design

To ensure our final design matches Zeplin's design, please add spacing  (right and left) of the form 🛠👍

image

Your design
image

Design on Zeplin
image


Kindly use the same font style and text color according to the given design 🤗

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @mohammadnewton 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Apply React language style guides in code

## Required Changes ♻️
_Please kindly check the comments under the review._

# Broken links ⛔
image

Please fix the broken links in the `Authors` section for a professional repository. 💪

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/styles.css [hunk 3]
```diff
@@ -1,5 +1,306 @@
...
+  color: #121212;
+  opacity: 0.5;
+}
+
+.circle {
+  margin-right: 3.46%;
+  border: 1px solid rgb(204, 200, 200);
+  border-radius: 50%;
+  padding: 0.5rem;
+}
```
Comment (KanzaTahreem) [inline src/styles/styles.css]: - [ ] Good job for adding a border 👏 However, [as suggested](https://github.com/mohammadnewton/bookstore/pull/5#discussion_r1306561098), please make sure that your design matches Zeplin's design. I noticed that the current border color is different from the one given to us. In the design guidelines, the border color is `e8e8e8`. Please fix this.

File: src/styles/styles.css [hunk 4]
```diff
@@ -1,5 +1,306 @@
...
+
+.hr {
+  border: 1px solid gray;
+  margin: 0 3rem;
+}
+
+.new-book {
+  padding: 0 3rem 0.5rem 3rem;
+  font-family: var(--RobotoSlab);
+  color: #a4a4a4;
```
Comment (KanzaTahreem) [inline src/styles/styles.css]: - [ ] Similarly here, [as suggested](https://github.com/mohammadnewton/bookstore/pull/5#discussion_r1306561142), please make sure to use the given text color which is `#888888`

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @mohammadnewton 

Great work on making the changes requested by a previous reviewer 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @mohammadnewton 

Good job implementing the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great
-  ✅ Great work using the right GitFlow 
-  ✅ Your README looks good
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nejidevelops** [comment]
Alhamdulillah I appreciate you I was super worried but thanks to you @KanzaTahreem

**Pull request merged**